deploy-manage/deploy/cloud-on-k8s/accessing-services.md

```diff
@@ -68,7 +68,9 @@ You can expose services in [different ways](https://kubernetes.io/docs/concepts/
 By default, the {{es}} service created by ECK is configured to route traffic to all {{es}} nodes in the cluster. Depending on your cluster configuration, you may want more control over the set of nodes that handle different types of traffic (query, ingest, and so on). Refer to [](./requests-routing-to-elasticsearch-nodes.md) for more information.
 
 ::::{warning}
-When you change the `clusterIP` setting of the service, ECK will delete and re-create the service as `clusterIP` is an immutable field. Depending on your client implementation, this might result in a short disruption until the service DNS entries refresh to point to the new endpoints.
+When you change the `clusterIP` setting of the service, ECK deletes and re-creates the service, as `clusterIP` is an immutable field. Depending on your client implementation, this might result in a short disruption until the service DNS entries refresh to point to the new endpoints.
+
+If you change the service’s `port`, set `targetPort` to the container’s default listening port. Otherwise, Kubernetes uses the same value for both, resulting in failed connections. For reference, default ports are `9200` for the {{es}} HTTP interface, and `5601` for {{kib}}.
```

Comment:
We could also include a list of reference default ports, like:
* 9200 for ES HTTP
* 9300 for ES Transport
* 9443 for ES RCS (Remote cluster server)
* 5601 for Kibana

But I didn't want to make the note too big. Ports 9300 and 9443 are mentioned in their relevant documents.

Reply:
I like "for example" - this makes me wonder if we have some sort of central resource we can link to for all of these ports though 🤔

Reply:
> this makes me wonder if we have some sort of central resource we can link to for all of these ports though 🤔

No, unfortunately we don't have that kind of reference documentation, we can create a doc issue though, I think it would make sense to have it in https://www.elastic.co/docs/deploy-manage/deploy/cloud-on-k8s/accessing-services